### ISSUE TITLE
Grow text downwards?

### ISSUE THREAD

**Issue by cannyboy** [hanspinckaers/GrowingTextView#71]
The GrowingTextView I want to use will be fixed to the top of the screen.

...so is there a way to grow the text downwards as they type?

**Comment by hanspinckaers** [comment]
Yes, you can change the code in the example to _not_ reposition the textview. It should grow downwards.
